dsl/pace/dsl/dace/orchestration.py

```diff
@@ -172,43 +183,80 @@ def build_sdfg(
     elif config.get_orchestrate() == DaCeOrchestration.BuildAndRun:
         MPI.COMM_WORLD.Barrier()
         if is_compiling:
-            unblock_waiting_tiles(comm, sdfg.build_folder)
+            unblock_waiting_tiles(MPI.COMM_WORLD, sdfg.build_folder)
```

Comment:
Is there ever a use case where we want to run dace backend with a null comm? Would it make sense to support that?

Reply:
Glad you asked :)

My goto way to test is to actually use NullComm to build the cache so that I only have one processor running and my machine doesn't die. This code is never reached in this case (hence MPI not being a problem). I will had a comment on top to explain why this is fine & what are the condition to keep alive so that this code doesn't blow up.

My personal recommendation would be to _never_ run 54 ranks for building Dace, but rather run 9 jobs with the NullComm from rank 0-8, so that you do the minimum amount of work in the minimum amount of time.